### DISCUSSION TITLE
Feature request: Adding offset to temperature

### DISCUSSION THREAD

**Discussion by marxzui** [Rem0o/FanControl.Releases#1229] [Ideas]
My CPU and GPU have different temperatures at which they start to throttle.

I would like my case fans to ramp up if either comes close to its temperature.

Currently, I don't know how to do that.

If it would be possible to add an offset to a temperature, I could target a single temperature.

Alternatively, an option to "sum" a fixed value to a "mix" virtual sensor would do the trick...

**Comment by Rem0o** [comment]
Create 1 graph per sensor, then mix them with a mix fan curve.

**Reply by marxzui** [reply]
Works well. Thank you!
